classes/db/DbPDO.php

```diff
@@ -365,13 +365,16 @@ public function getBestEngine()
 
         $sql = 'SHOW VARIABLES WHERE Variable_name = \'have_innodb\'';
         $result = $this->link->query($sql);
+
         if (!$result) {
```

Comment:
you can test it by adding on line 368 ``$result = false;``

without my PR => BOOOOM
with my PR => \o/\o/\o/